frontend/public/actions/ui.ts

```diff
@@ -16,6 +16,7 @@ import { allModels } from '../module/k8s/k8s-models';
 import { detectFeatures, clearSSARFlags } from './features';
 import { OverviewSpecialGroup } from '../components/overview/constants';
 import { setClusterID, setCreateProjectMessage, setUser, setConsoleLinks } from './common';
+import { Rule } from '../components/monitoring';
```

Comment:
Shared monitoring types have now moved to `public/components/monitoring/types.ts`